src/app/attendance/components/AttendanceFormBuilder.tsx

```diff
@@ -162,112 +165,123 @@ const AttendanceFormBuilder = () => {
+          action={() => form.handleSubmit(onSubmit)()}
+          className="max-w-lg mx-auto p-8 mt-10 bg-white shadow-md rounded-md space-y-6"
+        >
+          <h1 className="text-2xl font-bold text-center mb-6">
+            Attendance Form
+          </h1>
 
-        <FormField
-          control={form.control}
-          name="memberId"
-          render={({ field }) => (
-            <FormItem>
-              <FormLabel>User Name</FormLabel>
-              <FormControl>
-                <Select
-                  value={field.value?.toString() || ""}
-                  onValueChange={handleUserSelect}
-                  disabled={loginUser.role === "member"}
-                >
-                  <SelectTrigger
-                    className={`border rounded-md p-2 ${
-                      form.formState.errors.memberId
-                        ? "border-red-500"
-                        : "border-gray-300"
-                    }`}
+          <FormField
+            control={form.control}
+            name="memberId"
+            render={({ field }) => (
+              <FormItem>
+                <FormLabel>User Name</FormLabel>
+                <FormControl>
+                  <Select
+                    value={field.value?.toString() || ""}
+                    onValueChange={handleUserSelect}
+                    disabled={loginUser.role === "member"}
                   >
-                    <SelectValue placeholder="Select a user">
-                      {field.value
-                        ? roleBaseMembers.find((m) => m.id === field.value)
-                            ?.name
-                        : "Select a user"}
-                    </SelectValue>
-                  </SelectTrigger>
-                  <SelectContent>
-                    {roleBaseMembers.map((user) => (
-                      <SelectItem key={user.id} value={user.id.toString()}>
-                        {user.name}
-                      </SelectItem>
-                    ))}
-                  </SelectContent>
-                </Select>
-              </FormControl>
-              <FormMessage />
-            </FormItem>
-          )}
-        />
+                    <SelectTrigger
+                      className={`border rounded-md p-2 ${
+                        form.formState.errors.memberId
+                          ? "border-red-500"
+                          : "border-gray-300"
+                      }`}
+                    >
+                      <SelectValue placeholder="Select a user">
+                        {field.value
+                          ? roleBaseMembers.find((m) => m.id === field.value)
+                              ?.name
+                          : "Select a user"}
+                      </SelectValue>
+                    </SelectTrigger>
+                    <SelectContent>
+                      {roleBaseMembers.map((user) => (
+                        <SelectItem key={user.id} value={user.id.toString()}>
```

Comment:
Why `toString`?

Reply:
it expecting string 
![image](https://github.com/user-attachments/assets/8ecae5b4-a644-4191-ab5f-efeb18b9add7)

Reply:
`roleBaseMembers.map(x => ({...x, id: x.id.toString()}))`

Reply:
Fixed